roles/lmod/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+#
+# ansible role to install lmod:
+#
+---
+- name: "install packages"
+  become: yes
+  action: apt name=lmod state=latest
```

Comment:
Should use the extended format to be consistent with our other roles and the other tasks in this role